### ISSUE TITLE
OSB test

### ISSUE THREAD

**Issue by sunrise2399** [ygtxr1997/MSML#5]
Hello, using the pre-training model you provided and the model we trained ourselves, we could not correctly separate the scarfs, glasses on AR data set and masks on MFV data set. May I ask what causes it？

**Comment by ygtxr1997** [comment]
For AR dataset, you may need to finetune the OSB with its occlusions (labeling the occluded pixels by hand).
For MFV dataset, some masks truly cannot be detected by the OSB. 
The reason might be that our facial mask augmentation is too simple.
Using a stronger mask augmentation (increasing the number of masks or making the masked faces more natural) should alleviate this issue.
